src/app/components/tabs/index.html

```diff
@@ -142,6 +142,16 @@ <h3>Page titles</h3>
 			<p>
 				For information about how to use <a href="https://angular.io/docs/ts/latest/cookbook/set-document-title.html">the Angular <sky-code>Title</sky-code> service</a> and the SKY UX <sky-code>title</sky-code> configuration property to specify the title, see <a stacheRouterLink="learn/reference/configuration" fragment="app">the configuration reference</a>.
 			</p>
+
+			<h3>Tabs as site-level navigation</h3>
```

Comment:
Is this section supposed to be a child of the Page titles section? If so, I think this should be an h4 sine Page titles is an h3.

And speaking of the header, is there a reason it's not part of the includes file? That seems a little weird to me. But maybe it makes sense if we expect the header to be different sizes in different places.